hikari/impl/entity_factory.py

```diff
@@ -1184,8 +1182,8 @@ def deserialize_integration(
                 id=snowflakes.Snowflake(raw_application["id"]),
                 name=raw_application["name"],
                 icon_hash=raw_application["icon"],
-                summary=raw_application["summary"],
-                description=raw_application["description"],
+                summary=summary if (summary := raw_application["summary"]) else None,
+                description=description if (description := raw_application["description"]) else None,
```

Comment:
Wouldn't this be cleaner as `raw_application["field"] or None`

Reply:
Yep, will change